fideslog/api/routes/events.py

```diff
@@ -38,3 +39,30 @@ async def create(
         raise InternalServerError(err) from err
 
     return event
+
+
+@event_router.post(
+    "/user-registration",
+    response_description="The created event",
+    response_model=UserRegistrationEvent,
+    responses={
+        status.HTTP_429_TOO_MANY_REQUESTS: TooManyRequestsError.doc(),
+        status.HTTP_500_INTERNAL_SERVER_ERROR: InternalServerError.doc(),
+    },
+    status_code=status.HTTP_201_CREATED,
+)
+async def create_user_registration_event_route(
+    _: Request,
+    user_registration_event: UserRegistrationEvent,
+    database: Session = Depends(get_db),
+) -> UserRegistrationEvent:
+    """
+    Create a new user registration event.
+    """
+
+    try:
+        create_user_registration_event(database=database, event=user_registration_event)
+    except DBAPIError as err:
+        raise InternalServerError(err) from err
+
+    return user_registration_event
```

Comment:
Expanding on my above comment, it's unclear to me why this endpoint is necessary. We already have an endpoint for creating new `AnalyticsEvent` instances/database records, and the act of registering a user feels like something that can be handled through that endpoint. As written, we're not simply storing an event that corresponds to a user registration, we're instead storing an event that identifies the user that we've registered. This feels more appropriate for a `users` table in the core fides database, where it would be leveraged for access management.